doc/modules/linear_model.rst

```diff
@@ -786,8 +786,13 @@ non-smooth `penalty="l1"`. This is therefore the solver of choice for sparse
 multinomial logistic regression. It is also the only solver that supports
 `penalty="elasticnet"`.
 
-In a nutshell, the following table summarizes the penalties supported by
-each solver:
+The "lbfgs" is an optimization algorithm that approximates the 'BFGS'_, algortithm
+which belongs to quasi-Newton methods. It performs the best for small datasets
+then other solvers. The "lbfgs" solver is recommended for use for small data-sets
+but for larger datasets its performance suffers.
+`lbfgs performance <http://www.fuzihao.org/blog/2016/01/16/Comparison-of-Gradient-Descent-Stochastic-Gradient-Descent-and-L-BFGS/>`_.
```

Comment:
I think the words "lbfgs performance" alone will look a bit weird here. Are you trying to refer the reader to further study on this topic? To reference the source of your statements?

Reply:
Yes i am referencing the link for proof of lbfgs performance for small data sets.